src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line above the class declaration to adhere to the coding standards.